### ISSUE TITLE
dropout of 0.8 in mobilenetv3 paper

### ISSUE THREAD

**Issue by aragakiyuiii** [xiaolai-sqlai/mobilenetv3#32]
Thanks for your work!
In _mobilenetv3 paper 6.1.1 Training setup_, the **dropout rate is 0.8**. Do you use dropout in the code?
